### DISCUSSION TITLE
IMPORTANT Add a robots.txt to stop Google from indexing Deprecated Versions of the Docs

### DISCUSSION THREAD

**Discussion by aribornstein** [Lightning-AI/pytorch-lightning#6486] [Idea pool]
Google is currently indexing deprecated versions of the docs so that when you search for features you are getting depricated information. For example if I google "pytorch lightning save best model" The first result is associated with lighting version 0.4.9 with no indictation that isn't the latest version of the docs. 

![image](https://user-images.githubusercontent.com/3045407/110874149-e9fc3200-82db-11eb-8864-ed478c54e801.png)

This shouldn't be that hard to fix and will have major impact on developer experience. 
https://docs.readthedocs.io/en/latest/faq.html#how-can-i-avoid-search-results-having-a-deprecated-version-of-my-docs

**Comment by aribornstein** [comment]
https://github.com/PyTorchLightning/pytorch-lightning/issues/6488

**Reply by akihironitta** [reply]
#6160
